packages/boot/tools/supports.ts

```diff
@@ -367,6 +367,29 @@ export const makeSwingsetTestKit = async (
                   icaMocks.startChannelOpenInit.channelOpenAck(obj),
                 );
                 return undefined;
+              case 'sendPacket':
+                switch (obj.packet.data) {
+                  case 'eyJ0eXBlIjoxLCJkYXRhIjoiQ3FvQkNpTXZZMjl6Ylc5ekxuTjBZV3RwYm1jdWRqRmlaWFJoTVM1TmMyZEVaV3hsWjJGMFpSS0NBUUFBQUFBQUFBRUFBQUlBQUFBQUFnWUFBQUFBQUFVRUFBQUFCd0FBQUFBQUJ3QUFBQUFBQ0FBSUFBQUFBQUFHQUFZQUFBQUFBQUFBQUFBQUJnQUNCUUFIQUFJQUFBQUFBQUFBQUFBQUFBRUFBQUFBQUFBSUJnQUFBQUFBQWdBQUFBQUFCQUFBQUFBQ0FBQUFBQUFBQUFBSkF3QUVBd0FBQUFBQUFBQUFBQUFBQVFBPSIsIm1lbW8iOiIifQ==':
```

Comment:
Not sure this is the best way to mock different scenarios, but it's the only one I've come up with currently.

The structure of `obj`:

```js
 * @typedef {{
 *  method: 'sendPacket';
 *  packet: {
 *    data: string;
 *    destination_channel: ChannelId;
 *    destination_port: DestinationPort;
 *    source_channel: ChannelId;
 *    source_port: SourcePort;
 *  };
 *  relativeTimeoutNs: string;
 *  type: 'IBC_METHOD';
 * }} SendPacketMethod
 ```

Reply:
refactored to move these out to constants in `mocks.js`. [Discussion](https://github.com/Agoric/agoric-sdk/pull/9195#discussion_r1554459215). New success or failure cases can be added to this list, and referenced in the `sendPacket` switch case under `BridgeId.DIBC` in `support.ts`